.swagger-revision

```diff
@@ -1 +1 @@
-62fbcb9e8bec4a40569e51c7f316af85806628ee
+v2.2.0
```

Comment:
Yep. As it stands it checks out, cleans, and rebuilds on every execution. You can switch to a non-tag to fix this, or add the feature to `script/swagger`.